PluginsAndFeatures/azure-toolkit-for-intellij/azure-sdk-reference-book/src/main/java/com/microsoft/azure/toolkit/intellij/azuresdk/referencebook/AzureSdkArtifactGroupPanel.java

```diff
@@ -81,6 +101,7 @@ public void actionPerformed(@NotNull final AnActionEvent e) {
                 CopyPasteManager.getInstance().setContents(new StringSelection(viewer.getText()));
             }
         });
+        group.add(new DependencyTypeSelector(this::onDependencyTypeSelected));
```

Comment:
put maven/gradle selection above the copy button

Reply:
![image](https://user-images.githubusercontent.com/17075863/112397980-2600af80-8d3e-11eb-9451-ebe5e59730e9.png)

Reply:
Switch the button up/down sequence